src/lib/y2network/interfaces/bonding.rb

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/interfaces/virtual_interface"
+
+module Y2Network
+  module Interfaces
+    class Bonding < VirtualInterface
+      # Returns list of interfaces enslaved in the bond interface
+      #
+      # @return [Y2Network::InterfacesCollection] interfaces enslaved in the bond
+      # TODO: reading from backend should be handled elsewhere
```

Comment:
yeah, big topic. Should connection config knows it? As you can change slaves in UI, then I think using connection config makes sense for holding information about slaves. @teclator what do you think? Also what is order of dependency? connection_config depends on interface or vice versa?

Reply:
well, this was intended as temporary solution for isolating code from lanitems and using new API in the rest of the code. So, it is not perfect. I'd say that reading slaves should be in connections. In fact I'm slowly working on it as a part of https://trello.com/c/6CBQadMq/1146-2-infer-interfaces-type-correctly-when-reading-them however I don't have working prototype yet.